### PR TITLE
Add actionValidateOrderAfter

### PR DESCRIPTION
**Pull request by olecorre**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | PR#24738 Add a new Hook actionValidateOrderAfter, This hook is called after the complete creation of an order
| Fixed ticket? | Fixes #23787

### CODE REVIEW

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
thx @olecorre

File: src/content/1.7/modules/concepts/hooks/list-of-hooks.md [hunk 1]
```diff
@@ -1117,7 +1117,25 @@ Doesn't necessarily have to be paid.
...
+    <?php
+    array(
+      'cart' => (object) Cart,
+      'order' => (object) Order,
+      'customer' => (object) Customer,
+      'currency' => (object) Currency,
+      'orderStatus' => (object) OrderState
+    );
+    ```
+
```
Comment (PululuK) [inline src/content/1.7/modules/concepts/hooks/list-of-hooks.md:1138]: Available since ?

**Comment by olecorre** [comment]
The PR has just been merged, I have been asked to complete the doc

File: src/content/1.7/modules/concepts/hooks/list-of-hooks.md (hunk 1 above)
Comment (kpodemski) [reply src/content/1.7/modules/concepts/hooks/list-of-hooks.md:1138]: whoops :)

**Comment by kpodemski** [comment]
@olecorre can you do it? :)

**Comment by PululuK** [comment]
Hi @olecorre and thanks you for your contribution !
I'm talking about `Available since` tag. Example :
image

The new hook, will be available from which version?

**Comment by PululuK** [comment]
> @olecorre can you do it? :)

Nop ! See https://github.com/PrestaShop/PrestaShop/pull/24172#pullrequestreview-650213390
I will discuss with the team on slack.

Thanks @olecorre

**Pull request merged**